doc/_utils/utils.py

```diff
@@ -117,8 +117,17 @@ def add_google_analytics(app: Sphinx) -> None:
         app: Sphinx instance.
     """
 
-    app.add_js_file("https://www.googletagmanager.com/gtag/js?id=G-ZPVZRKFQJR")
-    app.add_js_file("js/ga-tracker.js")
+    app.add_js_file(
+        filename=None,
+        type="text/javascript",
+        body="""
+        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        })(window,document,'script','dataLayer','GTM-WF4CVFX');
+        """
+    )
```

Comment:
can't you create a js file in the static folder and just include it?

Reply:
fixed, thanks!